src/main.ts

```diff
@@ -0,0 +1,82 @@
+import {getInput, info, saveState, setFailed} from '@actions/core'
+import {exec} from '@actions/exec'
+import {join} from 'path'
+import {tmpdir} from 'os'
+import {promises} from 'fs'
+import {wait} from './wait'
+import optionMappingJson from './option-mapping.json'
+
+const CONTAINER_VERSION = '4.6.2'
```

Comment:
Making this an GH Action option would be great. Let's do this in a separate PR though.